.github/ISSUE_TEMPLATE/new_rule.yaml

```diff
@@ -0,0 +1,88 @@
+name: New Rule
+description: Suggestions and ideas for new rules
+title: "[New Rule] Name of rule"
+labels: ["Rule: New", "Team: TRADE"]
+assignees: []
+projects: ["elastic/1268"]
+
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: "Provide a detailed description of the activity to be detected."
+      placeholder: "Detailed description..."
+
+  - type: dropdown
+    id: target_ruleset
+    attributes:
+      label: Target Ruleset
+      description: "Select the target rulset."
+      options:
+        - apm
+        - cross-platform
+        - integrations
+        - linux
+        - macos
+        - ml
+        - network
+        - promotions
+        - threat_intel
+        - windows
+        - other
+
+  - type: dropdown
+    id: rule_type
+    attributes:
+      label: Target Rule Type
+      description: "Select the target type."
+      options:
+        - KQL
+        - ML_Job
+        - Threshold
+        - EQL
+        - Indicator_Match
+        - New_Terms
+        - ESQL
+
+  - type: input
+    id: ecs_version
+    attributes:
+      label: Tested ECS Version
+      description: "Specify the tested ECS version."
+      placeholder: "x.x.x"
+
+  - type: textarea
+    id: query
+    attributes:
+      label: Query
+      description: "Provide the query for the rule (optional)."
+      placeholder: "Query..."
+
+  - type: textarea
+    id: new_fields
```

Comment:
Not sure what we may be asking specifically for here. Do we expect community members to know which fields are new and which are not based on the schemas that we build locally? Likely unknown fields will be identified during unit tests.

Reply:
Good question. It might be helpful if they know. I just pulled this from the older `new_rule.md` file. Not sure how much it was used.

Reply:
++ https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#about-yaml-syntax-for-issue-forms 

> Note: The required field key is only supported in public repositories. In private and internal repositories, all fields are optional.